src/Rules/Node/ForbiddenFunctionNodeVisitor.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Rules\Node;
+
+use Twig\Environment;
+use Twig\Node\Expression\FunctionExpression;
+use Twig\Node\Node;
+use TwigCsFixer\Rules\AbstractNodeVisitorRule;
+
+final class ForbiddenFunctionNodeVisitor extends AbstractNodeVisitorRule
+{
+    /**
+     * @param list<string> $forbidden
+     */
+    public function __construct(
+        private array $forbidden,
```

Comment:
```suggestion
        private array $forbiddenFunctions,
```
or just `functions` ?